### PR TITLE
Protractor e2 e tests

### PR DESCRIPTION
**Pull request by trevorhays**
Created 2 tests to test logging in through the UI.  
1.  Valid username and password.  Asserts that the user is directed to the dashboard page
2.  Bad username and password.  Asserts that the message "Invalid username or password." is displayed.

### CODE REVIEW

**Comment by trevorhays** [comment]
Applies to issue. https://github.com/apache/incubator-trafficcontrol/issues/1597

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by mitchell852** [comment]
@trevorhays - this looks good but what do you think about moving these tests to:

traffic_portal/test/browser_automation <-- not sure if browser_automation is the best term for it but E2E makes me think these tests somehow test our system E2E...

also, by putting it under traffic_portal/test maybe we end up adding other sorts of tests at some point like

traffic_portal/test/browser_automation
traffic_portal/test/unit

**Comment by mitchell852** [comment]
actually, maybe e2e makes sense so how about 

traffic_portal/test/end_to_end or traffic_portal/test/e2e

**Comment by trevorhays** [comment]
Sounds good.  I'll change it to traffic_portal/test/end_to_end.

File: traffic_portal/test/end_to_end/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# UI Tests  (Using Protractor)
+
+- Install and run protractor
+  http://www.protractortest.org/#/tutorial
+
+- Make sure Traffic Portal is running
+
+- Edit conf.js if necessary to match your environment.
+
```
Comment (mitchell852) [inline traffic_portal/test/end_to_end/README.md]: How about adding a step here like 

Start up Selenium Server

webdriver-manager start

**Comment by trevorhays** [comment]
Sure.  I'll add that in.

**Pull request merged**